io.github.sameboy.SameBoy.yml

```
@@ -0,0 +1,52 @@
+app-id: io.github.sameboy.SameBoy
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
+build-options:
+  append-path: /usr/lib/sdk/llvm13/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm13/lib
+command: sameboy
+rename-desktop-file: sameboy.desktop
+rename-icon-file: sameboy
```

it's 
```suggestion
rename-icon: sameboy
```

there is even a warning about that.